assets/json/fundable_oss.json

```diff
@@ -10,15 +10,15 @@
     "id": 19,
     "name": "ffxiv-teamcraft/ffxiv-teamcraft",
     "url": "https://github.com/ffxiv-teamcraft/ffxiv-teamcraft",
-    "stars": 364,
-    "stars_display": "364"
+    "stars": 372,
+    "stars_display": "372"
```

Comment:
Hi sorry for the late reply. I have request:

**Update:**
- ffxiv-teamcraft/ffxiv-teamcraft - 374 stars